src/libtiled/maptovariantconverter.cpp

```diff
@@ -228,6 +228,9 @@ QVariant MapToVariantConverter::toVariant(const Tileset &tileset,
         tilesetVariant[QStringLiteral("imageheight")] = tileset.imageHeight();
     }
 
+    if (tileset.transformationFlags()!=Tileset::NoTransformation)
```

Comment:
I prefer just `if (tileset.transformationFlags())`, which is clear enough in meaning no flags are set. I needed the `Tileset::NoTransformation` entry mostly for initialization purposes.